format/spec.md

```diff
@@ -418,7 +416,7 @@ Engines may model operations as deleting/inserting rows or as modifications to r
 
 This example demonstrates how `_row_id` and `_last_updated_sequence_number` are assigned for a snapshot. This starts with a table with a `next-row-id` of 1000.
 
-Writing a new append snapshot would create snapshot metadata with `first-row-id` assigned to the table's `next-row-id`:
+Writing a new append snapshot creates snapshot metadata with `first-row-id` assigned to the table's `next-row-id`:
```

Comment:
Is this limited to `append`? I would think any snapshot (including rewrite) should assign ids.